caab-service/src/main/java/uk/gov/laa/ccms/caab/api/entity/Opponent.java

```diff
@@ -38,18 +44,106 @@ public class Opponent implements Serializable {
   private Long id;
 
   @Embedded
-  private AuditTrail auditTrail;
+  private AuditTrail auditTrail = new AuditTrail();
 
   @ManyToOne
   @JoinColumn(name = "FK_APPLICATION", nullable = false)
   private Application application;
 
-  @Column(name = "RELATIONSHIP_TO_CASE", length = 50)
-  private String relationshipToCase;
+  @Column(name = "EBS_ID", length = 50)
+  private String ebsId;
 
   @Column(name = "TYPE", length = 50)
   private String type;
 
-  //TO BE CONTINUED IN FURTHER STORY
+  @Column(name = "TITLE", length = 50)
```

Comment:
All of these attributes should probably have a javadoc

Reply:
TODO

Reply:
Done